src/components/accordion-search.tsx

```diff
@@ -83,6 +84,10 @@ type AccordionSearchItemProps = {
    * The user's query, passed to highlight in the item's label
    */
   query: string;
+  /**
+   * Indicates if the item has more values to return on request (in case of xrefs)
+   */
+  addAsterix?: boolean;
```

Comment:
```suggestion
  addAsterisk?: boolean;
```
I had to google this one as i've never had to spell the word "asterisk"!

Reply:
My bad. I was so confident this was the correct spelling. Apparently, it is a comic character.